cibuild.cmd

```diff
@@ -52,7 +52,9 @@ echo Building this commit:
 git show --no-patch --pretty=raw HEAD
 
 REM Restore the NuGet packages 
-.\run.cmd sync || goto :BuildFailed
+.\run.cmd sync 
+echo Exit Code is %errorlevel%
```

Comment:
I know, this is debug code.  cibuild.cmd just stops running after running `run.cmd sync`.  I thought it was because the sync failed, but after checking in this change, it doesn't even get to the echo.